### PR TITLE
feat: Improve buckets on mobile

### PR DESCRIPTION
**Pull request by ajdaniel**
For mobile, make the buckets a bit easier to read. Because scrolling is easy on mobile, just render all of the buckets at their full height. 

If you'd rather a max height on each bucket, just set the height/max-height property in the css. If going for a max height, I'd recommend `100vh`

### CODE REVIEW

File: public/components/training/training.css [hunk 1]
```diff
@@ -17,6 +17,17 @@
     min-height: 200px;
 }
 
+@media (max-width: 550px) {
+    .trainingbuckets {
+        height: initial;
```
[the remark this answers is not in the document]
Comment (ajdaniel) [reply public/components/training/training.css]: It’s a good way to unset variables,  clever!

**Comment by dalelane** [comment]
@ajdaniel I realised this morning that she probably wouldn't have been able to log in at all before Saturday morning, as that was when https://github.com/IBM/taxinomitis/commit/f0b462000384758b95e86d69b11e6960de928863 went live. So it was spooky timing anyway :-)

**Comment by ajdaniel** [comment]
Weird!

**Comment by ajdaniel** [comment]
By the way, I wasn't able to run this locally because I think my set of env vars is out of date, so I haven't technically tested this. I just used the chrome dev tools while rendering a mobile version of the site.

**Pull request merged**